django/forms/fields.py

```diff
@@ -601,6 +602,7 @@ def has_changed(self, initial, data):
 
 
 class ImageField(FileField):
+    default_validators = [validate_image_file]
```

Comment:
You're right. I forgot that `field.to_python()` is the first validation step. I will revert this change.